databricks/koalas/indexing.py

```diff
@@ -82,6 +82,80 @@ def _unfold(key, kseries):
     return rows_sel, cols_sel
 
 
+class AtIndexer(object):
+    """
+    Access a single value for a row/column label pair.
+    Similar to ``loc``, in that both provide label-based lookups. Use ``at`` if you only need to
+    get a single value in a DataFrame or Series.
```

Comment:
you should also mention the behavior when multiple values are matched (returning an array)

Reply:
I've added it to the docs with 0dc9e08.